graphql/image_test.go

```diff
@@ -60,3 +64,85 @@ func TestToolchains(t *testing.T) {
 	require.NotNil(t, res[0])
 	assert.Equal(t, testToolchain, utility.FromStringPtr(res[0].Name))
 }
+
+func TestDistros(t *testing.T) {
+	setupPermissions(t)
+	require.NoError(t, db.ClearCollections(distro.Collection), "unable to clear distro collection")
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestDistros")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+	d1 := &distro.Distro{
+		Id:      "ubuntu1604-large",
+		ImageID: "ubuntu1604",
+	}
+	require.NoError(t, d1.Insert(ctx))
+	d2 := &distro.Distro{
+		Id:      "ubuntu1604-small",
+		ImageID: "ubuntu1604",
+	}
+	require.NoError(t, d2.Insert(ctx))
+	d3 := &distro.Distro{
+		Id:      "rhel82-small",
+		ImageID: "rhel82",
+	}
+	require.NoError(t, d3.Insert(ctx))
+	imageID := "ubuntu1604"
+	image := model.APIImage{
+		ID: &imageID,
+	}
+	res, err := config.Resolvers.Image().Distros(ctx, &image)
+	require.NoError(t, err)
+	require.Len(t, res, 2)
+	distroNames := []string{}
+	distroNames = append(distroNames, *res[0].Name, *res[1].Name)
```

Comment:
nit. use `utility` functions. Also I think you can do this (declare rather than append):
```suggestion
	distroNames := []string{ /item 1/, /item 2/ }
```